app/models/manager_refresh/inventory_collection/scanner.rb

```diff
@@ -0,0 +1,19 @@
+module ManagerRefresh
+  class InventoryCollection
+    class Scanner
```

Comment:
Yes, a class for every problem? :-D I might move this elsewhere, when scanning will turn to more generic preprocessing.